Scripts/General/ModMercs.lua

```diff
@@ -0,0 +1,732 @@
+            end
+        end
+        
+        Table[Counter].Level = {}
+        for i = 0, 3 do
+            if string.len(Words[i+15]) > 0 then
+                Table[Counter].Level[i+1] = tonumber(Words[i+15])
+            end
+        end
+
+        Table[Counter].Attack1 = {}
+        for i = 0, 3 do
+            if string.len(Words[i+19]) > 0 then
+                Table[Counter].Attack1[i+1] = tostring(Words[i+19])
+            end
+        end
+
+        Table[Counter].Attack2 = {}
+        for i = 0, 3 do
+            if string.len(Words[i+23]) > 0 then
+                Table[Counter].Attack2[i+1] = tostring(Words[i+23])
+            end
+        end
+
+        Table[Counter].FightsMax = {}
+        for i = 0, 3 do
+            if string.len(Words[i+27]) > 0 then
+                Table[Counter].FightsMax[i+1] = tonumber(Words[i+27])
+            end
+        end
+
+        Counter = Counter + 1
+	end
+
+	io.close(File)
+
+    -- Credentials
+    FilePath    = "Data/Tables/MercCredentials.txt"
+	File	    = io.open(FilePath)
+	if not File then
+		return
+	end
+
+	LineIt = File:lines()
+	LineIt()
+
+    Counter = 1
+
+    -- Process escape sequences
+    local function procEsc(str)
+        str = str:gsub("\\n", "\n")    -- Convert \n to newline
+        return str
+    end
+
+	for line in LineIt do
+        
+        local Words = string.split(line, "\9")
+		if string.len(Words[1]) == 0 then
+			break
+		end
+
+		Table[Counter].Credentials = table.copy(SMercCredentials)
+
+		if string.len(Words[3]) > 0 then
+			Table[Counter].Credentials.PriceHire = tonumber(Words[3])
+		end
+        if string.len(Words[4]) > 0 then
+			Table[Counter].Credentials.PriceReHire = tonumber(Words[4])
+		end
+        if string.len(Words[5]) > 0 then
+			Table[Counter].Credentials.PriceResurrect = tonumber(Words[5])
+		end
+        if string.len(Words[6]) > 0 then
+			Table[Counter].Credentials.TextAbout = procEsc(tostring(Words[6]))
+		end
+        if string.len(Words[7]) > 0 then
+			Table[Counter].Credentials.TextHired = procEsc(tostring(Words[7]))
+		end
+        if string.len(Words[8]) > 0 then
+			Table[Counter].Credentials.TextGreeting = procEsc(tostring(Words[8]))
+		end
+        if string.len(Words[9]) > 0 then
+			Table[Counter].Credentials.TextFightTired = procEsc(tostring(Words[9]))
+		end
+        if string.len(Words[10]) > 0 then
+			Table[Counter].Credentials.TextSpecial = procEsc(tostring(Words[10]))
+		end
+        if string.len(Words[11]) > 0 then
+			Table[Counter].Credentials.TextFireAttempt = procEsc(tostring(Words[11]))
+		end
+        if string.len(Words[12]) > 0 then
+			Table[Counter].Credentials.TextFireCancel = procEsc(tostring(Words[12]))
+		end
+
+        Counter = Counter + 1
+    end
+
+    io.close(File)
+end
+
+-- Mercenaries
+ParseMercTables(MercsDB) -- load from Datatable at "Data/Tables/<>.txt"
+for i, Merc in ipairs(MercsDB) do
+    MercNPCList[i] = Merc.NPC_ID
+end
+
+-- EXAMPLE DECLARATION
```

Comment:
Yay 🎊 example usage. Should this be at the bottom, beginning or maybe in separate file?